### ISSUE TITLE
Gradle issue

### ISSUE THREAD

**Issue by smaples23** [react-native-camera/react-native-camera#3471]
The Android Gradle plugin supports only kotlin-android-extensions Gradle plugin version 1.6.20 and higher.
The following dependencies do not satisfy the required version:
project ':react-native-vision-camera' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.30

**Comment by awamalik** [comment]
Have you got any Solution to this?

**Comment by smaples23** [comment]
> Have you got any Solution to this?

No, I have since switched to building the app on 8thwall. I came across too many deprecated dependencies doing it this way
